guacamole/src/main/webapp/app/settings/services/preferenceService.js

```diff
@@ -58,7 +58,7 @@ angular.module('settings').factory('preferenceService', ['$injector',
          *
          * @type String
          */
-        inputMethod : null
+        inputMethod : 'none'
```

Comment:
If not now, then when? I doubt we'll have a user story like "replace disgusting use of string literals with constants".

I'll just do it now...